File: src/main/java/team4/footwithme/chat/service/response/ChatMemberResponse.java
package team4.footwithme.chat.service.response;

import team4.footwithme.chat.domain.ChatMember;

public record ChatMemberResponse(
    Long chatroomId,
    Long memberId
) {
    public ChatMemberResponse(ChatMember chatMember) {
        this(
            chatMember.getChatRoom().getChatroomId(),
            chatMember.getMember().getMemberId()
        );
    }
}

File: src/main/java/team4/footwithme/chat/service/response/ChatMemberResponse.kt
package team4.footwithme.chat.service.response

import team4.footwithme.chat.domain.ChatMember

@JvmRecord
data class ChatMemberResponse(
    val chatroomId: Long?,
    val memberId: Long?
) {
    constructor(chatMember: ChatMember) : this(
        chatMember.chatRoom!!.chatroomId,
        chatMember.member!!.memberId
    )
}
